src/components/Chart/Donut.tsx

```diff
@@ -81,9 +97,19 @@ const StyledContent = styled.div`
   vertical-align: middle;
 `
 
-const Donut = ({
-  height,
-  width,
+type DonutProps = {
+  chartId?: string
+  content?: React.ReactNode
+  data: Data[]
+  focused?: number
+  height?: number
+  onFocusChange(...args: unknown[]): unknown
```

Comment:
FocusEventHandler maybe ?

Reply:
No it's a bit like the `onChange` on Stepper, it's a custom made one.

I updated though